wikimedia/wikipedia-ios
# [YiR] Prompt user to log in after survey

**Phabricator:** https://phabricator.wikimedia.org/T376350

### Notes
This PR presents an alert view to prompt the user to log in after dismissing the survey. It navigates to the login flow using our existing LoginCoordinator.

Note: I am trying to reuse very similar existing strings we had for the button titles, so the copy doesn't exactly match the Figma. I will run this through design review.

Since this task was a nice-to-have for V1, I am doing the specific translations and metrics separately. This way we don't have to consider it as we wrap up [T376666](https://phabricator.wikimedia.org/T376666) and [T376302](https://phabricator.wikimedia.org/T376302).

### Test Steps
1. Fresh install app. Ensure primary app language is target wiki. Take steps to ensure entry point appears while logged out (build up article history, go to Settings, disable YiR, then enable YiR, background/foreground).
2. Go through year in review feature. After dismissing survey after the last slide, login prompt should appear.
